lib/openstack_query/mappings/server_mapping.py

```diff
@@ -30,6 +34,19 @@ class ServerMapping(MappingInterface):
     Define property mappings, kwarg mappings and filter function mappings, and runner mapping related to servers here
     """
 
+    @staticmethod
+    def get_chain_mappings():
+        """
+        Should return a dictionary containing property pairs mapped to query mappings.
+        This is used to define how to chain results from this query to other possible queries
+        """
+        return {
+            ServerProperties.USER_ID: UserProperties.USER_ID,
+            ServerProperties.PROJECT_ID: ProjectProperties.PROJECT_ID,
+            ServerProperties.FLAVOR_ID: FlavorProperties.FLAVOR_ID
+            # TODO hypervisor mapping, image mapping etc
```

Comment:
Nit: drop this TODO, since we'll get it anyway as we need the other forms of chaining